bundles/net.tourbook.statistics/src/net/tourbook/statistics/graphs/StatisticTooltipUI_CategorizedData.java

```diff
@@ -761,9 +761,6 @@ private void updateUI() {
       final float elevationUp_Percentage                 = elevationUp_Summary == 0 ? 0 : elevationUp / elevationUp_Summary   * 100;
       final float numTours_Percentage                    = numTours_Summary    == 0 ? 0 : numTours    / numTours_Summary      * 100;
 
-      final float elevationUp_WithMeasurement            = elevationUp / UI.UNIT_VALUE_ELEVATION;
```

Comment:
@wolfgang-ch 
I think I found a bug in the statistics. My statistics being wrong, I investigated and when I changed the unit system in the preferences, it seems that the values from the array `_tourData_Common.elevationUp_High_Resorted` already come converted in the appropriate unit. Maybe you will know better since you worked on that recently and maybe you will know if there are more bugs in other types of statistics.
Before the fix, the elevation gain number in the TourBook View is not the same in the statistics
![image](https://user-images.githubusercontent.com/25235665/120090178-ab496d00-c0bd-11eb-88a3-065792cdee9e.png)

After my fix, the numbers are good and match what I see in the TourBookView
![image](https://user-images.githubusercontent.com/25235665/120090140-632a4a80-c0bd-11eb-8a9b-160055cfbc5e.png)

Reply:
The data are adjusted to the measurement system when they are loaded from the database, so elevation was twice adjusted.

https://github.com/wolfgang-ch/mytourbook/blob/c8d6d02b7697f4cbca8aff85b283c462e692f32a/bundles/net.tourbook.statistics/src/net/tourbook/statistics/graphs/DataProvider_Tour_Month.java#L232-L235

Reply:
I found the same bug in the file `StatisticTooltipUI_TourFrequency.java.`
I have fixed it in this [PR](https://github.com/wolfgang-ch/mytourbook/pull/351)